lib/discourse-publish.php

```diff
@@ -112,21 +170,11 @@ public function publish_post_after_save( $post_id, $post ) {
 			$post_time             = strtotime( $post->post_date );
 
 			if ( ( ( 0 === $force_publish_max_age ) || $post_time >= $min_date ) && $is_rest_request ) {
-				$force_publish_post = true;
 				update_post_meta( $post_id, 'publish_post_category', intval( $this->options['publish-category'] ) );
```

Comment:
@scossar To keep this method focused on a single boolean outcome (i.e. force_publish) perhaps we should move this post meta update outside into the `publish_post_after_save` method, conditional on force_publish?